pkgs/development/python-modules/backgroundremover/default.nix

```diff
@@ -0,0 +1,77 @@
+{ buildPythonPackage
+, certifi
+, charset-normalizer
+, fetchPypi
+, ffmpeg-python
+, filelock
+, filetype
+, gdown
+, hsh
+, idna
+, lib
+, more-itertools
+, moviepy
+, numpy
+, pillow
+, pymatting
+, pysocks
+, requests
+, scikitimage
+, scipy
+, six
+, torch
+, torchvision
+, tqdm
+, urllib3
+, waitress
+}:
+
+buildPythonPackage rec {
+  pname = "backgroundremover";
+
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-hh+NSFy2NgvgpsnTcFORzFfW4ZxX96nP4u9O+OpZw7Y=";
+  };
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    ffmpeg-python
+    filelock
+    filetype
+    gdown
+    hsh
+    idna
+    more-itertools
+    moviepy
+    numpy
+    pillow
+    pymatting
+    pysocks
+    requests
+    scikitimage
+    scipy
+    six
+    torch
+    torchvision
+    tqdm
+    urllib3
+    waitress
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "BackgroundRemover is a command line tool to remove background from image and video, made by nadermx to power https://BackgroundRemoverAI.com";
```

Comment:
Ah thanks @alyssais ! So yes it could be utilised as both a library as well as a binary, would you prefer I move it into the correct location (top-level/all-packages.nix) as the initial commit or is it okay to exist as a new commit?
Further to this, I note `hsh` falls into the same category so I'll resolve for that also

Reply:
I think you have it the wrong way round — if it's both, keep it as is, but add a `toPythonApplication` line in all-packages.nix.  Please make the changes in the commits that add each package.